site/docs/components/overlay/usage.mdx

```diff
@@ -45,12 +45,12 @@ import { Overlay, OverlayTrigger, OverlayPanel } from "@salt-ds/lab";
 
 ### `Overlay`
 
-<PropsTable packageName="lab" componentName="Overlay" />
+<PropsTable packageName="core" componentName="Overlay" />
 
 ### `OverlayTrigger`
 
-<PropsTable packageName="lab" componentName="OverlayTrigger" />
+<PropsTable packageName="core" componentName="OverlayTrigger" />
```

Comment:
OverlayTrigger and OverlayPanel appear to have no props. If it's the case, should we include the empty table?

cc @mikearildbrown

Reply:
https://saltdesignsystem-git-overlay-to-core-fed-team.vercel.app/salt/components/overlay/usage#props